data/localized/de_patch/text/conversations/06_stronghold/06_cv_visitor_criminal.stringtable

```diff
@@ -206,46 +206,46 @@ Nyry macht auf dem Absatz kehrt und verlässt schnellen Schrittes den Saal.</Def
     </Entry>
     <Entry>
       <ID>39</ID>
-      <DefaultText>"Ja ... ja, gut. Es war nicht leicht, mich immer verstecken zu müssen. Vielleicht ist es besser, neu anzufangen. Danke, dass Sie mir die Chance dazu geben."</DefaultText>
+      <DefaultText>„Ja&#160;… ja, gut. Es war nicht leicht, mich immer verstecken zu müssen. Vielleicht ist es besser, neu anzufangen. Danke, dass Sie mir die Chance dazu geben.“</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>40</ID>
-      <DefaultText>"Kommt nicht in Frage! Sie sind zwar gerissen, aber Sie bringen mich nicht dazu, dem Henker meinen Hals freiwillig anzubieten."
+      <DefaultText>„Kommt nicht in Frage! Sie sind zwar gerissen, aber Sie bringen mich nicht dazu, dem Henker meinen Hals freiwillig anzubieten.“
 
 Nyry macht auf dem Absatz kehrt und verlässt schnellen Schrittes den Saal.</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>41</ID>
-      <DefaultText>"Kommt nicht in Frage! Sie sind zwar gerissen, aber Sie bringen mich nicht dazu, dem Henker meinen Hals freiwillig anzubieten."
+      <DefaultText>„Kommt nicht in Frage! Sie sind zwar gerissen, aber Sie bringen mich nicht dazu, dem Henker meinen Hals freiwillig anzubieten.“
 
 Nyry macht auf dem Absatz kehrt und verlässt schnellen Schrittes den Saal.</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>42</ID>
-      <DefaultText>"Ja ... ja, gut. Es war nicht leicht, mich immer verstecken zu müssen. Vielleicht ist es besser, neu anzufangen. Danke, dass Sie mir die Chance dazu geben."</DefaultText>
+      <DefaultText>„Ja&#160;… ja, gut. Es war nicht leicht, mich immer verstecken zu müssen. Vielleicht ist es besser, neu anzufangen. Danke, dass Sie mir die Chance dazu geben.“</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>43</ID>
-      <DefaultText>"Sind Sie [Player Name]? Ich könnte ein wenig Hilfe gebrauchen."</DefaultText>
+      <DefaultText>„Sind Sie [Player Name]? Ich könnte ein wenig Hilfe gebrauchen.“</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>44</ID>
-      <DefaultText />
+      <DefaultText>„Ich bitte Sie natürlich nicht, das umsonst zu tun. Ich habe hier ein paar Sachen abgeholt von einem&#160;… Bekannten. Sie nutzen mir nicht viel, wenn ich erwischt werde.“</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>45</ID>
-      <DefaultText />
+      <DefaultText>„Burgvogt, bitte die Schmelztiegelritter, Gnade walten zu lassen.“</DefaultText>
```

Comment:
Wir haben das überall sowohl in der männlichen, als auch der weiblichen Form. Also beide.